docs/stories/03-inputs/Combobox.stories.tsx

```diff
@@ -189,6 +190,64 @@ export const Creatable: Story = {
   },
 };
 
+export const CreatableAlwaysVisible: Story = {
+  render: () => {
+    const [value, setValue] = React.useState<string | undefined>('');
+
+    return (
+      <Combobox
+        aria-describedby="creatable combobox"
+        placeholder="My favourite fruit is..."
+        value={value}
+        onChange={setValue}
+        creatable
+        createItemAlwaysVisible
```

Comment:
I would like to avoid adding a new property for that. This will turn into a props hell in the future. 

Can't we do something like this ⬇️  
```suggestion
        creatable="sticky"
```

So "creatable" could be typed as `creatable: boolean | 'sticky'` to avoid breaking changes, handle our new sticky use case and still be flexible in the future.

Reply:
I decided to use "visible" because at the end we want to show the create button always visible